pkg/secrets/secrets.go

```diff
@@ -101,13 +101,14 @@ func getHostSecretData(hostDir string) ([]secretData, error) {
 func getMounts(filePath string) []string {
 	file, err := os.Open(filePath)
 	if err != nil {
-		logrus.Warnf("file %q not found, skipping...", filePath)
+		// This is expected on most systems
+		logrus.Debugf("file %q not found, skipping...", filePath)
 		return nil
 	}
 	defer file.Close()
 	scanner := bufio.NewScanner(file)
 	if err = scanner.Err(); err != nil {
-		logrus.Warnf("error reading file %q, skipping...", filePath)
+		logrus.Errorf("error reading file %q, skipping...", filePath)
```

Comment:
Should include the error in here, so we know what went wrong (permissions, etc)

Reply:
Fixed